src/handleApps.ts

```diff
@@ -27,15 +27,16 @@ export async function handleApps(
 
   const proxyResponse = await fetch(proxyRequest);
 
+  const finalResponse = new Response(proxyResponse.body, proxyResponse);
   const additionalHeaders = corsHeaders(request, orgSettings);
   for (let additionalHeader in additionalHeaders) {
-    proxyResponse.headers.append(
```

Comment:
It's not valid to append headers to an existing response. You must create a new response instead. See https://developers.cloudflare.com/workers/examples/modify-response/